starboard/common/BUILD.gn

```diff
@@ -81,7 +81,6 @@ static_library("common") {
     "reuse_allocator_base.h",
     "rwlock.cc",
     "rwlock.h",
-    "scoped_ptr.h",
```

Comment:
Can you also try to remove `starboard/common/move.h` in the same PR ? I believe `scoped_ptr` is the only file still including it.

Reply:
done